src/gui/cookiesdialog.cpp

```diff
@@ -85,5 +87,16 @@ void CookiesDialog::onButtonAddClicked()
 
 void CookiesDialog::onButtonDeleteClicked()
 {
-    m_cookiesModel->removeRow(m_ui->treeView->selectionModel()->currentIndex().row());
+    QModelIndexList idxs = m_ui->treeView->selectionModel()->selectedRows();
+
+    // sort in descending order
+    std::sort(idxs.begin(), idxs.end(),
```

Comment:
Don't understand why do you sort indices here. Does it matter in what order they are removed?